app/code/Tasks/FeedbackManager/Controller/Adminhtml/Manage/Decline.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Adminhtml\Manage;
+
+use Magento\Backend\App\Action;
+use Tasks\FeedbackManager\Helper\Sendmail;
+use Magento\Backend\App\Action\Context;
+
+class Decline extends Action
+{
+    protected $feedbackFactory;
+    protected $mailer;
+    
+    public function __construct(
+        Context $context,
+        \Tasks\FeedbackManager\Model\FeedbackFactory $feedbackFactory,
+        Sendmail $mailer
+    ) {
+        $this->feedbackFactory = $feedbackFactory;
+        $this->mailer = $mailer;
+        parent::__construct($context);
+    }
+
+    public function execute()
+    {
+        $resultRedirect = $this->resultRedirectFactory->create();
+        $id = $this->getRequest()->getParam('id');
+        
+        if (isset($id)) {
+            
+            try {
+                $feedbackModel = $this->feedbackFactory->create();
+                $feedbackModel->load($id);
+                $feedbackModel->setData('status', 2)
+                        ->save();
+                
+                $mailStatus = $this->mailer->sendMail("feedback_decline_template" , (array)$feedbackModel->getData());
+                if ($mailStatus) { $this->messageManager->addSuccessMessage(__('Customer notified on his/her feedback decline status.')); }
```

Comment:
@gayathri-aravind Don't write inline statements in PHP. It'll hinder readability

Reply:
Inline statements are removed and code structured. Commit: 882571dc9e3988e3e1f109a2d06fe316213a2971